benchmarks/data_struct_benchmarks/multi_value_dict/benchmark.py

```diff
@@ -0,0 +1,32 @@
+import copy
+
+from django.utils.datastructures import MultiValueDict
+
+from ...utils import bench_setup
+
+
+class MultiValueDictBench:
+    def setup(self):
+        bench_setup()
+        self.case = {"a": ["a"], "b": ["a", "b"], "c": ["a", "b", "c"]}
+        self.update = {"a": ["a"], "b": ["a", "b"], "c": ["a", "b", "c"]}
+
+    def time_multi_value_dict(self):
+        caseDict = MultiValueDict(self.case)
```

Comment:
```suggestion
        case_dict = MultiValueDict(self.case)
```

Let's use snake_case.